js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -26,46 +26,96 @@ import {FIFTEEN_SECONDS, useRefreshAtInterval} from '../app/QueryRefresh';
 import {currentPageAtom} from '../app/analytics';
 import {PythonErrorFragment} from '../app/types/PythonErrorFragment.types';
 import {AssetGroupSelector} from '../graphql/types';
+import {useUpdatingRef} from '../hooks/useUpdatingRef';
 import {PageLoadTrace} from '../performance';
 import {useBlockTraceUntilTrue} from '../performance/TraceContext';
-import {useIndexedDBCachedQuery} from '../search/useIndexedDBCachedQuery';
+import {fetchPaginatedData} from '../runs/fetchPaginatedBucketData';
+import {CacheManager} from '../search/useIndexedDBCachedQuery';
 import {LoadingSpinner} from '../ui/Loading';
 
 type Asset = AssetTableFragment;
 
 const groupTableCache = new Map();
 
+const ASSET_CATALOG_TABLE_QUERY_VERSION = 1;
+const BATCH_LIMIT = 100000;
```

Comment:
We can lower this limit once we take the limit into account on the backend, but for now 100k seems to work for January-capital

Reply:
We do take the limit into account... I think ideally this batch limit is something like 1000?

Reply:
oh you're right, we only do in OSS.  I can put up a PR